rules_gapic/bazel/src/main/java/com/google/api/codegen/bazel/ArgsParser.java

```diff
@@ -29,15 +33,37 @@
               + "The required arguments are: "
               + required;
       System.out.println(msg);
+      ArgsParser.printUsage();
       throw new IllegalArgumentException(msg);
     }
   }
 
+  static void printUsage() {
+    String helpMessage =
+        "Usage (when running from googleapis folder):\n"
+            + "  bazel run //:build_gen -- --src=rules_gapic/bazel/src/test/data/googleapis\n"
```

Comment:
Now the help became specific to googleapis context... That was kind of the reason why I did nto want to include any helps at all. Notice, the tool in its help speaks about how it must be executed from bazel (the java tool does not have to be bazel specific).